spec/helpers/image_helper_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+describe ImageHelper, type: "helper" do
+  let(:path) { "a/b/c" }
+  let(:image_double) { instance_double(Image, build_args: {}, alt: "xyz") }
+
+  describe "#image_args" do
+    before do
+      allow(Image).to receive(:new).and_return(image_double)
+      image_args(path)
+    end
+
+    specify "passes the path to Image#build_args" do
+      expect(image_double).to have_received(:build_args).with(path).once
+    end
+  end
+
+  describe "#alt" do
```

Comment:
It feels a bit odd that the expectation isn't on the args/alt text for these tests - this reads better in my mind but I don't feel strongly about it (also noticed a typo in the `#alt` describe, should be `#image_alt`):

```
let(:path) { "a/b/c" }
let(:image_double) { instance_double(Image, build_args: { some: "args" }, alt: "xyz") }

before { allow(Image).to receive(:new).and_return(image_double) }

describe "#image_alt" do
  it { expect(image_alt).to eq(image_double.alt) }
end

describe "#image_args" do
  it { expect(image_args(path)).to eq(image_double.alt) }
end
```